security-analysis/security-analysis-api/src/main/java/com/powsybl/security/PostContingencyComputationStatus.java

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security;
+
+/**
+ * @author Bertrand Rix <bertrand.rix at artelys.com>
+ */
+public enum PostContingencyComputationStatus {
```

Comment:
If we use the same for post contingency status and post action status what could be the best name?
Is PostContingencyComputationStatus still ok?
@sylvlecl ?